.changeset/empty-islands-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+"effect": patch
```

Comment:
```suggestion
"effect": minor
```

Reply:
Fixed. FYI I picked "patch" based on this README instruction, not sure if the pre-release mode part is still accurate in the 3.0 era.

https://github.com/Effect-TS/effect/blob/c12d7ff1935f74cf8d3c3e8e9396dd68dd52a56f/README.md?plain=1#L54